sqlglot/dialects/dialect.py

```diff
@@ -46,6 +49,31 @@ class Dialects(str, Enum):
     Doris = "doris"
 
 
+class NormalizationStrategy(str, Enum):
+    """Specifies the strategy according to which identifiers should be normalized."""
+
+    LOWERCASE = "lowercase"  # Unquoted identifiers are lowercased
+    UPPERCASE = "uppercase"  # Unquoted identifiers are uppercased
+    CASE_SENSITIVE = "case_sensitive"  # Always case-sensitive, regardless of quotes
+    CASE_INSENSITIVE = "case_insensitive"  # Always case-insensitive, regardless of quotes
+
+    @property
+    def is_lowercase(self):
```

Comment:
i don't like this pattern of properties, just use the enums

Reply:
As in `<my_enum>.value == ...`? It seems a little too verbose :P

Reply:
no, you can just check strategy is Strategy.LOWERCASE etc

Reply:
Ok